hc/front/tests/test_add_pushbover.py

```diff
@@ -37,4 +37,6 @@ def test_it_validates_nonce(self):
         r = self.client.get("/integrations/add_pushover/?%s" % params)
         assert r.status_code == 403
 
-    ### Test that pushover validates priority
+#    ### Test that pushover validates priority
```

Comment:
If this code is not being run it's good practice to removed it, rather than commenting it out